cpp/src/arrow/engine/substrait/extension_set.cc

```diff
@@ -519,6 +519,21 @@ struct ExtensionIdRegistryImpl : ExtensionIdRegistry {
     return maybe_converter->second;
   }
 
+  Result<SubstraitCallToArrow> GetSubstraitCallToArrowFallback(
+      util::string_view function_name) const override {
```

Comment:
should we do?
```suggestion
      const util::string_view& function_name) const override {
```

Reply:
`string_view` is effectively a "pointer" (well, a pointer and a size) and it is small enough that it isn't worth passing by reference: https://quuxplusone.github.io/blog/2021/11/09/pass-string-view-by-value/